### PR TITLE
Assignment8 MattJett-EWU & twofingerrighclick

### PR DESCRIPTION
**Pull request by twofingerrightclick**
Working with @MattJett-EWU

### CODE REVIEW

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -1,35 +1,89 @@
...
+                        <li><a asp-controller="Users" asp-action="Index">Users</a></li>
+                        <li><a asp-controller="Groups" asp-action="Index">Groups</a></li>
+                        <li><a asp-controller="Gifts" asp-action="Index">Gifts</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </section>
+
+    <div id="hello">
```
Comment (MattJett-EWU) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml]: awesome job on the template!

File: SecretSanta/src/SecretSanta.Web/scripts/app.ts [hunk 2]
```diff
@@ -49,5 +60,35 @@ export module App {
...
+
+        }
+
+        async deleteGroups() {
+
+        }
+
+        async getGroups() {
+
+        }
```
Comment (MattJett-EWU) [inline SecretSanta/src/SecretSanta.Web/scripts/app.ts]: not sure if these were needed or not.

File: SecretSanta/src/SecretSanta.Web/Views/Gifts/Index.cshtml [hunk 3]
```diff
@@ -1,31 +1,3 @@
...
-                    <a asp-action="Edit" asp-route-id="@item.Id">Edit</a> |
-                    <a asp-action="Delete" asp-route-id="@item.Id">Delete</a>
-                </td>
-            </tr>
-        }
-    </tbody>
-</table>
\ No newline at end of file
+﻿<section class="section">
+    <ul id="giftList"></ul>
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/Views/Gifts/Index.cshtml]: ul is probably not the right item to use here, could have just been a div instead. a ul expects someone to add li elements to it, but that isn't the case here.

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -1,35 +1,89 @@
...
-    <div>
+
+    <section class="hero is-primary is-small is-family-secondary is-bold">
+        <!-- Hero head: will stick at the top -->
+        <div class="hero-head">
+            <nav class="navbar">
+                <div class="container">
+                    <div class="navbar-brand">
+                        <a class="navbar-item">
+                            <img src="https://bloximages.newyork1.vip.townnews.com/news-journal.com/content/tncms/assets/v3/editorial/9/d9/9d9d2aca-4c42-11ea-9659-0fcffa7c6907/5e41bb507a581.image.jpg?resize=620%2C264" alt="Logo">
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml]: nice addition :)

**Pull request closed without merging**